index.html

```diff
@@ -73,6 +79,272 @@ <h1>I’m Sebastian. Glad to see you!</h1>
+          contact me.
+        </p>
+
+        <div class="connect-div">
+          <a href="#" class="connect-a">
+            <h3>LET'S CONNECT</h3>
+          </a>
+        </div>
+
+        <div class="social-icons about-ms">
+          <ul>
+            <li>
+              <a href="#"><i class="fa fa-instagram"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"> <i class="fa fa-github"></i> </a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin"></i></a>
+            </li>
+          </ul>
+        </div>
+
+        <button class="see-proj-btn-itself resume-btn" type="button">
+          Get my resume
+        </button>
+      </div>
+
+      <div class="about-second-section intro">
+        <ul class="accordion-menu">
+          <li>
+            <div class="dropdownlink">
+              <h3 class="about-titles" id="Language-title">Languages</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"></i>
+            </div>
+            <ul class="submenuItems">
+              <li>
+                <img
+                  src="./styling-features-pictures/JS-CIRCLE.png"
+                  alt=""
+                  class="circle-menu-pic"
+                />
+                <a href="#">JavaScript</a>
+              </li>
+              <li>
+                <img
+                  src="./styling-features-pictures/HTML-CIRCLE.png"
+                  alt=""
+                  class="circle-menu-pic"
+                />
+                <a href="#">HTML</a>
+              </li>
+              <li class="no-margin-bottom">
+                <img
+                  src="./styling-features-pictures/CSS-CIRCLE.png"
+                  alt=""
+                  class="circle-menu-pic"
+                />
+                <a href="#">CSS </a>
+              </li>
+            </ul>
+          </li>
+
+          <li>
+            <div class="dropdownlink bottom-separator">
+              <h3 class="about-titles" id="fr-title">Frameworks</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"></i>
+            </div>
+            <!-- <ul class="submenuItems">
+                <li><a href="#">Fiction book 1</a></li>
+                <li><a href="#">Fiction book 2</a></li>
+                <li><a href="#">Fiction book 3</a></li>
+              </ul> -->
+          </li>
+
+          <li>
+            <div class="dropdownlink bottom-separator">
+              <h3 class="about-titles" id="sk-title">Skills</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"> </i>
+            </div>
+            <!-- <ul class="submenuItems">
+                <li><a href="#">Fantasy book 1</a></li>
+                <li><a href="#">Fantasy book 2</a></li>
+                <li><a href="#">Fantasy book 3</a></li>
+              </ul> -->
```

Comment:
- [ ] To make your code clear and clean, please kindly, remove the commented codes in your repo.